composer.json

```diff
@@ -1,18 +1,35 @@
 {
-    "name": "solaris/php-moon-phase",
-    "description": "A PHP class for calculating the phase of the Moon.",
-    "homepage": "http://www.github.com/solarissmoke/php-moon-phase",
-    "keywords": ["php", "moon", "phase"],
-    "type": "library",
-    "license": "MIT",
-    "authors": [
-        { "name": "Samir Shah", "email": "samir@rayofsolaris.net" },
-        { "name": "Erik Scheepers", "email": "e.scheepers@linkorb.com", "role": "Packaging for composer" }
-    ],
-    "require": {
-        "php": ">=5.3.0"
-    },
-    "autoload": {
-        "psr-0": { "Solaris": "" }
-    }
+	"name": "janbarasek/php-moon-phase",
```

Comment:
Don't think you meant to change the name and homepage of the package?

Reply:
Ou, sorry! It changed it because I want make new package for internal use. In new pull-request commit it will be reverted.